### DISCUSSION TITLE
How to pass data from table row to a form

### DISCUSSION THREAD

**Discussion by othAs-dev** [Budibase/budibase#9308] [Help]
Hi,
I want to pass data from a table row to a form which is in another url.
I access to the id when I passed it in the url but the infos of my row like the first name or other information I can't to access it.
I access to my data by a queries I have a mysql database, so I can't use auto screen.
I'm asking for your help, thank you all 🙂

**Comment by melohagan** [comment]
Hey @othAs-dev 

I'm not entirely sure what you are asking. Have you had a look at this docs page? https://docs.budibase.com/docs/url-parameters
